packages/block-library/src/image/image.js

```diff
@@ -274,6 +274,21 @@ export default function Image( {
 		}
 	}
 
+	function resetLightbox() {
+		// When deleting a link from an image while lightbox settings
+		// are enabled by default, we should disable the lightbox,
+		// otherwise the resulting UX looks like a mistake.
```

Comment:
This one is actually a pretty common case! If at any time, a user has the following in their `theme.json`...
```
{
	"$schema": "https://schemas.wp.org/trunk/theme.json",
	"version": 2,
	"settings": {
		"blocks": {
			"core/image": {
				"lightbox": {
					"allowEditing": true,
					"enabled": true
				}
			}
		}
	}
}
```
...and also has a link configured for the image, then one potential way to handle that UX would be to show the popup for the link, and if that link is removed, then immediately show the popup for the lightbox. However, I found that UX to be confusing.

Here I go over more in depth what the issue is. Let me know what you think, and I'm also happy to discuss more or answer any questions 😄

https://github.com/WordPress/gutenberg/assets/5360536/32065bda-d229-43ed-8655-3af4c5efffb2

Reply:
ok, I understand it now. Thank you for the explanation! 

I agree that this is a better UX than just "jumping" to the lightbox. 👍 

Could you add a link to this PR in this very comment in the code (line 280)? It might be helpful for someone in the future to refer to your video explanations.

Reply:
Done 🙂